editor/editor_node.cpp

```diff
@@ -4807,14 +4807,19 @@ void EditorNode::_update_recent_scenes() {
 	Array rc = EditorSettings::get_singleton()->get_project_metadata("recent_files", "scenes", Array());
 	recent_scenes->clear();
 
-	String path;
-	for (int i = 0; i < rc.size(); i++) {
-		path = rc[i];
-		recent_scenes->add_item(path.replace("res://", ""), i);
-	}
+	if (rc.size() <= 0) {
```

Comment:
`Array::size()` can't be negative, so this should check `== 0`. (It's the same outcome obviously, but `<= 0` size checks are inconsistent with how we do it throughout the codebase.)